pkg/grafana/templating/templates/grafana-user-values.yaml.template

```diff
@@ -1,4 +1,8 @@
 grafana:
   grafana.ini:
+    auth:
+      disable_signout_menu: false
     auth.generic_oauth:
```

Comment:
Log out is necessary because the config takes some.time to load (with the operator work and so on) so we need to bé able to force a relogin. This might bé going away in the future if we enabled persistance but that still need to bé tested

Reply:
How disabling the signout menu has something to do with the config load time ?

Reply:
So what happens is that whenever there's a change on the grafana user values, grafana gets recreated and the operator creates the datasources and the organizations. But this is a bit slow (1-5 minutes) so if someone logs in before it's done, then we get a token with missing permissions. In that case, the only solution is to log out then in again

Reply:
I think we should somehow prevent the Grafana pod from being ready and allowing user to login before its fully configured.

Reply:
But then if the pod is not ready, the operator will only bé able to call the one that is ready which is the old pod

Reply:
Better solution would bé to enable persistance and not care about pod restarts iml

Reply:
Oh right that's tricky then. But still I do not get why disabling the signout menu helps here

Reply:
We're not disabling, we're enabling it because we disabled it in shared config

Reply:
Right now we cannot log out

Reply:
With this, we can logout to force the token refresh by logging again. Otherwise we have to wait an hour until thé currently jwt token is expired

Reply:
oh right disable=false, double negation is hard ^^

Ok fine then, let's go with this.